### ISSUE TITLE
Change Request: Don't Warn That Files Were Ignored by Default When They Were Ignored Explicitly

### ISSUE THREAD

**Issue by Kurt-von-Laven** [eslint/eslint#15035]
### Environment

Environment Info:

Node version: v16.8.0
npm version: v7.21.0
Local ESLint version: Not found
Global ESLint version: Not found
Operating System: linux 5.11.0-34-generic

My local ESLint version is v7.32.0, and I do not have ESLint globally installed. I believe my local ESLint version was not detected properly by `yarn run eslint --env-info`, because I am using [Yarn v2 Zero-Installs](https://yarnpkg.com/features/zero-installs) and hence don't have a `node_modules` directory.

### What parser are you using?

@typescript-eslint/parser

### What did you do?

```markdown
Create an empty file named `.foo`. Run `eslint --max-warnings 0 --ignore-path .foo`.

<details>
<summary>Configuration</summary>
root: true
parser: "@typescript-eslint/parser"
plugins: ["@typescript-eslint"]
extends: [
    "eslint:recommended",
    "plugin:@typescript-eslint/recommended",
    # See https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md#usage-with-prettier
    "eslint-config-prettier",
  ]
</details>
```

### What did you expect to happen?

```markdown
I expected the command to succeed without warning. The usual suggestion would be not to pass .foo to eslint if you desire to ignore it, but that is [tedious to implement](https://github.com/okonet/lint-staged#eslint--7-1) (c.f., "How can I ignore files from .eslintignore?") in this case. We have a lint-staged configuration which runs ESLint as a pre-commit hook whenever the version of Node.js specified in .nvmrc changes.
```

### What actually happened?

```markdown
ESLint found too many warnings (maximum: 0).

/home/kurt/Documents/ScribeMD/scribemd-server/.nvmrc
  0:0  warning  File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern '!<relative/path/to/filename>'") to override

✖ 1 problem (0 errors, 1 warning)
```

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by nzakas** [comment]
Duplicate of https://github.com/eslint/eslint/issues/15010
